assets/dev/js/frontend/handlers/container/grid-container.js

```diff
@@ -24,12 +26,14 @@ export default class GridOutline extends elementorModules.frontend.handlers.Base
 			outlineParentContainer: null,
 			gridOutline: this.findElement( selectors.gridOutline ),
 			directChildGridOverlay: this.findElement( selectors.directGridOverlay ),
+			emptyView: this.findElement( selectors.emptyView )[ 0 ],
+			container: this.$element[ 0 ],
```

Comment:
@davseve Where is `container` used?

Reply:
@davseve I am asking `where` is it being used?

Reply:
I removed the container from the elements object I prefer to use the raw value
`this.$element[ 0 ]`

Reply:
Here:
https://github.com/elementor/elementor/blob/4f9e7420e24a2c93794779a06a6a401a11891a0c/assets/dev/js/frontend/handlers/container/grid-container.js#L250